packages/mui-material/src/Alert/Alert.js

```diff
@@ -202,8 +204,13 @@ const Alert = React.forwardRef(function Alert(inProps, ref) {
             title={closeText}
             color="inherit"
             onClick={onClose}
+            {...componentsProps.closeIconButton}
           >
-            <CloseIcon fontSize="small" />
+            {components.CloseIcon ? (
```

Comment:
```suggestion
            {CloseIcon ? (
```

Extract this in a const above, to be consistent with how it is done in other components. Also, for consistency I think we should keep the types and resolving of the props as it is done in the unstyled components.